ConsoleApp/Generics/Program.cs

```diff
@@ -0,0 +1,120 @@
+﻿using System;
+
+namespace ConsoleApp;
+
+class Program
+{
+    static private Vote _vote;
```

Comment:
for future - access modifier should come first
```suggestion
    private static Vote _vote;
```